foris/locale/de/LC_MESSAGES/foris.po

```diff
@@ -587,10 +589,10 @@ msgid ""
 "all your devices. It usually has less interference, but the signal does not "
 "carry so well indoors."
 msgstr ""
-"Das 2,4 GHz Frequenzband ist unter den Clientgeräten breiter unterstützt, es "
-"ist aber mehr durch Interferenzstörungen belastet. Das 5 GHz Frequenzband "
```

Comment:
I never came across the term before, and was under the impression it was a error in the translation. I mean the word "Interferenz" alone implies a disturbance of a service, and the Duden defines it in regards to physics as the overlapping of two or more waves, which is quite fitting for the problem here. ([http://www.duden.de/rechtschreibung/Interferenz](http://www.duden.de/rechtschreibung/Interferenz)) :)